tests/Doctrine/Tests/DBAL/Functional/Schema/SchemaManagerFunctionalTestCase.php

```diff
@@ -1360,4 +1361,43 @@ public function commentsProvider() : array
             'valid with extra opening brackets' => ['(DC2Type:should((.stop)).before)', 'should((.stop', $currentType],
         ];
     }
+
+    public function testCreateAndListSequences() : void
+    {
+        if ( ! $this->_sm->getDatabasePlatform()->supportsSequences()) {
+            $this->markTestSkipped('This test is only supported on platforms that support sequences.');
+        }
+
+        $sequence1Name           = 'sequence_1';
+        $sequence1AllocationSize = 1;
+        $sequence1InitialValue   = 1;
+        $sequence2Name           = 'sequence_2';
+        $sequence2AllocationSize = 2;
+        $sequence2InitialValue   = 2;
+        $sequence1               = new Sequence($sequence1Name, $sequence1AllocationSize, $sequence1InitialValue);
+        $sequence2               = new Sequence($sequence2Name, $sequence2AllocationSize, $sequence2InitialValue);
+
+        $this->_sm->createSequence($sequence1);
+        $this->_sm->createSequence($sequence2);
+        /** @var Sequence[] $actualSequences */
+        $actualSequences = [];
+        foreach ($this->_sm->listSequences() as $sequence) {
+            $actualSequences[$sequence->getName()] = $sequence;
+        }
+
+        $isOracle      = $this->_sm->getDatabasePlatform()->getName() === OraclePlatform::NAME;
```

Comment:
Oracle is not the only supported platform which returns upper-cased column names. DB2 is the other one but it looks DBAL doesn't support sequences in DB2. In any event, checking the platform name is not the best approach.

I can think of two solutions:

1. Figure out why the rest of the tests in this class don't fail on Oracle without using the portability layer. For instance, `testListTableColumns()` expects column names to be lower-cased, however, they are upper-cased in Oracle. Must be implemented in:
https://github.com/doctrine/dbal/blob/22844d3c0d52e44c312ad60571b3d9e8da662c9b/lib/Doctrine/DBAL/Schema/AbstractSchemaManager.php#L824
2. Use `Portability\Connection` with `PORTABILITY_FIX_CASE` and `fetch_case = CASE_LOWER` when instantiating the schema manager. This is hor a real production code would deal with the case discrepancy.

Reply:
`testListTableColumns` also converts table name to lowercase `self::assertEquals('test', strtolower($columns['test']->getname()));`

Can do it this way then. In https://github.com/doctrine/dbal/blob/22844d3c0d52e44c312ad60571b3d9e8da662c9b/lib/Doctrine/DBAL/Schema/AbstractSchemaManager.php#L824 it just creates an array with lowercased table names as keys.

Reply:
It would be a breaking change. Doing `strtolower()` in a test would mean that the production code should do the same by design which it not true, it's a bug.

Let's skip the test for Oracle for now and fix the schema manager in 3.0.